src/main/java/com/google/ehub/data/EntertainmentItemBuilder.java

```diff
@@ -0,0 +1,71 @@
+package com.google.ehub.data;
+
+import java.util.Optional;
+
+public final class EntertainmentItemBuilder {
```

Comment:
If you add this builder as a static inner class of EntertainmentItem.java, then you can make the constructor of EntertainmentItem private so that nobody will use it. 

Something like:
`public static final class Builder { .... }`

Then reference the builder via `EntertainmentItem.Builder`

Reply:
OK I added the builder as a static class inside EntertainmentItem in last commit.